app.js

```diff
@@ -34,3 +34,52 @@ var stuff = [
 //   // console.log(stuff[i].name);
 //   // console.log(stuff[i].src);
 // }
+
+function findRandom(){
+  var tempArray = [];
+  while(tempArray.length < 3){
+    var rand = stuff[Math.floor(Math.random() * stuff.length)].src;
+    if(tempArray.indexOf(rand) === -1){
+      tempArray.push(rand);
+    }
+    // console.log(tempArray);
+  }
+  randomNumArray = tempArray;
+}
+findRandom();
+console.log(randomNumArray);
+
+var pictureBlockEl = document.getElementById('productPictures');
+function postPictures(){
+  var pictureEl = document.createElement('h1');
```

Comment:
Is this a picture element? Or is it a header element? 

When you have a set of repeated items in no particular order, what element do you normally put it into?